packages/editor/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@serlo/editor",
-  "version": "0.22.2",
+  "version": "0.23.0",
```

```suggestion
  "version": "0.23.0-beta.0",
```

I have the feeling we should first check whether it looks good on the NPM site (e.g is the string `AGPL-3.0-or-later` the correct one. I would first publish a release candidate and when everything is alright we publish `0.23.0`

---

Sounds good to me.